Catappult/appcoins-wallet-android
# [task/ESKL-132] Cleanup generated schemas

**What does this PR do?**

Self-explanatory

**Database changed?**

   No

**Where should the reviewer start?**

N/A

**How should this be manually tested?**

Well, make a build and confirm if the schemas are correctly generated, but not added to git.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ESKL-132](https://aptoide.atlassian.net/browse/ESKL-132)

**Questions:**

N/A



**Code Review Checklist**
